### ISSUE TITLE
False negative on nested CSS

### ISSUE THREAD

**Issue by ai** [fpetrakov/stylelint-gamut#14]
CSS:

```css
html {
  --hover: oklch(58% 0.22 260 / 20%);

  @media (color-gamut: p3) {
    --hover: oklch(54% 0.27 260 / 0.2);
  }
}

::selection {
  background: var(--hover);
}
```

Error:

```
> stylelint index.css


index.css
 10:15  ✖  Unexpected color out of gamut range       gamut/color-no-out-gamut-range
           "var(--hover)"
```

But P3 `--hover` is wrapped to `@media (color-gamut: p3)`. Unwrapped version [is not P3](https://oklch.evilmartians.io/#57.99999999999999,0.22,260,20).

Reproduction: https://github.com/ai/-stylelint-gamut-bug

**Comment by fpetrakov** [comment]
Thanks for the contribution! I will try to fix it asap

**Comment by fpetrakov** [comment]
I've published new version with bug fix, go check it out and report any issues :)

**Comment by ai** [comment]
Thanks. Now it works.
